pt-BR/mail/chrome/messenger/searchresults.dtd

```diff
@@ -1,33 +1,33 @@
 <!-- rich search results -->
-<!ENTITY searchresults.showthumbnail.label         "Show Thumbnails">
+<!ENTITY searchresults.showthumbnail.label         "Exibir miniaturas">
 <!ENTITY searchresults.showthumbnail.accesskey     "S">
-<!ENTITY viewByRelevance.label                     "Sorted By Relevance">
-<!ENTITY viewByDate.label                          "Sorted By Date">
-<!ENTITY searchresults.viewByRelevance.label       "Sort By Relevance">
-<!ENTITY searchresults.viewByDate.label            "Sort By Date">
-<!ENTITY addToCalendar.label                       "Add to Calendar">
-<!ENTITY prevNextResults.label                     "Prev / Next">
-<!ENTITY searchResultsNoMatchesFound.desc          "No items matched your search.">
-<!ENTITY searchResultsNoMatchesSuggestionAllFolders.desc          "Try searching in All Folders.">
-<!ENTITY searchResultsNoMatchesSuggestionAllFocus.desc          "Try setting your Focus to All.">
-<!ENTITY searchResultsNoMatchesSuggestionAllMessages.desc       "Try setting your view to Messages.">
-<!ENTITY showSearchOptions.label                   "Show search options">
+<!ENTITY viewByRelevance.label                     "Ordenadas por relevância">
+<!ENTITY viewByDate.label                          "Ordenadas por data">
+<!ENTITY searchresults.viewByRelevance.label       "Ordenar por relevância">
+<!ENTITY searchresults.viewByDate.label            "Ordenar por data">
+<!ENTITY addToCalendar.label                       "Adicionar ao Calendário">
+<!ENTITY prevNextResults.label                     "Anterior / Próximo">
+<!ENTITY searchResultsNoMatchesFound.desc          "Nenhum item corresponde à sua pesquisa.">
+<!ENTITY searchResultsNoMatchesSuggestionAllFolders.desc          "Tente pesquisar em Todas as Pastas.">
+<!ENTITY searchResultsNoMatchesSuggestionAllFocus.desc          "Tente alterar para Todos.">
+<!ENTITY searchResultsNoMatchesSuggestionAllMessages.desc       "Tenta alterar a exibição para Mensagens.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY searchResultsNoMatchesSuggestionAllMessages.desc       "Tente alterar a exibição para a vista de Mensagens.">